docs/source/tutorial_bt.rst

```diff
@@ -66,150 +18,115 @@ First, we do the necessary imports and some matplotlib style choices.
 
-We can get the internal representation of the |LTIModel| `fom`
 
-.. jupyter-execute::
-
-    fom
+Balanced truncation
+-------------------
 
-From this, we see that the matrices were wrapped in |NumpyMatrixOperators|,
-while default values were chosen for :math:`D` and :math:`E` matrices
-(respectively, zero and identity). The operators in an |LTIModel| can be
-accessed directly, e.g., `fom.A`.
+As the name suggests,
+the balanced truncation method consists of
+finding a *balanced* realization of the full-order LTI system and
+*truncating* it to obtain a reduced-order model.
 
-We can also see some basic information from `fom`'s string representation
-
-.. jupyter-execute::
-
-    print(fom)
-
-To visualize the behavior of the `fom`, we can draw its magnitude plot, i.e.,
-a visualization of the mapping :math:`\omega \mapsto H(\imath \omega)`, where
-:math:`H(s) = C (s E - A)^{-1} B + D` is the transfer function of the system.
-
-.. jupyter-execute::
-
-    w = np.logspace(-2, 8, 50)
-    _ = fom.mag_plot(w)
-
-We can also see the Bode plot, which shows the magnitude and phase of the
-components of the transfer function.
-In particular, :math:`\lvert H_{ij}(\imath \omega) \rvert` is in subplot
-:math:`(2 i - 1, j)` and :math:`\arg(H_{ij}(\imath \omega))` is in subplot
-:math:`(2 i, j)`.
-
-.. jupyter-execute::
-
-    _ = fom.bode_plot(w)
-
-Plotting the Hankel singular values shows us how well an LTI system can be
-approximated by a reduced-order model
-
-.. jupyter-execute::
-
-    hsv = fom.hsv()
-    fig, ax = plt.subplots()
-    ax.semilogy(range(1, len(hsv) + 1), hsv, '.-')
-    _ = ax.set_title('Hankel singular values')
-
-As expected for a heat equation, the Hankel singular values decay rapidly.
+The balancing part is based on the fact that a single LTI system has many
+realizations.
+For example, starting from a realization
 
+.. math::
 
-Running balanced truncation
----------------------------
+    \begin{align}
+        E \dot{x}(t) & = A x(t) + B u(t), \\
+        y(t) & = C x(t) + D u(t),
+    \end{align}
 
-The balanced truncation method consists of finding a balanced realization of the
-full-order LTI system and truncating it to obtain a reduced-order model. In
-particular, there exist invertible transformation matrices
-:math:`T, S \in \mathbb{R}^{n \times n}` such that the equivalent full-order
-model with
-:math:`\widetilde{E} = S^T E T = I`,
-:math:`\widetilde{A} = S^T A T`,
-:math:`\widetilde{B} = S^T B`,
-:math:`\widetilde{C} = C T`
-has Gramians :math:`\widetilde{P}` and :math:`\widetilde{Q}`, i.e., solutions to
-Lyapunov equations
+another realization can be obtained by replacing :math:`x(t)` with
+:math:`T \tilde{x}(t)` or by pre-multiplying the differential equation with an
+invertible matrix.
+In particular, there exist invertible transformation matrices
+:math:`T, S \in \mathbb{R}^{n \times n}` such that the realization with
+:math:`\tilde{E} = S^{\operatorname{T}} E T = I`,
```

Comment:
This only holds if E is invertible right? Maybe it should be mentioned that we consider an LTI system with invertible E matrix at the beginning of the paragraph.